### DISCUSSION TITLE
Unable to jog router

### DISCUSSION THREAD

**Discussion by animasguitars** [winder/Universal-G-Code-Sender#1680] [Help]
I have been using UGS without any problem sending my post processor from Fusion 360 to my Shapeoko XL. Now it will not jog. It does the homing, but when I go to jog it to my project it just makes a noise. The numbers in the X, Y or Z change as if the router is moving, but it stays stationary. I also use "Inventables" and "Carbide Motion". The jog works in both.

**Comment by breiler** [comment]
Are you using a jog feed rate higher than your machine is cable of moving?

**Comment by animasguitars** [comment]
Don't think so. The rate is "1", I have never changed it.

**Reply by breiler** [reply]
The rate is might be set too low then, have you tried turning it up. 
For instance, if you are using metric units that would mean that it would move 1 mm/min.

**Comment by animasguitars** [comment]
That was the problem. I turned the rate up and now it works perfectly
again. Thank-you so much. Obviously I am quite new to this and appreciate
your help
